gcd/gcd.schema.json

```diff
@@ -25,8 +25,12 @@
       "description": "SAID of the credential",
       "type": "string"
     },
+    "u": {
+      "description": "A salty nonce",
+      "type": "string"
+    },
     "i": {
-      "description": "Identifier of the issuer (the delegator)",
+      "description": "AID of the issuer(the delegator)",
```

Comment:
Please add a space before the paren. Leaving off the space is non-grammatical English.

Reply:
updated as per comment